kafka-connect/kafka-connect-runtime/src/integration/java/org/apache/iceberg/connect/TestContext.java

```diff
@@ -51,6 +52,7 @@ public class TestContext {
   private TestContext() {
     ComposeContainer container =
         new ComposeContainer(new File("./docker/docker-compose.yml"))
+            .withStartupTimeout(Duration.ofMinutes(2))
```

Comment:
How is this timeout related to test failures like `ExceptionInInitializerError`?

Reply:
My hope is this will give more time for the containers to initialize before failing and retrying.

Reply:
If containers don't initialize in time, the tests can continue to run and fail eventually?

Reply:
It looked like the tests were failing to initialize the containers before running.

Reply:
> It looked like the tests were failing to initialize the containers before running.

yeah that's also what I was seeing when I looked into this a bit

Reply:
Can we fail early when the containers are not initialized? The current exceptions look really confusing.

Reply:
I moved the context init to `BeforeAll` so the test will fail at initialization, which should be more clear.